templates/zerver/api/add-code-playground.md

```diff
@@ -27,6 +27,8 @@
 
 {generate_return_values_table|zulip.yaml|/realm/playgrounds:post}
 
+{generate_response_description(/realm/playgrounds:post)}
```

Comment:
It appears the organization you've gone with is this:
```
## Parameters

{generate_api_arguments_table|zulip.yaml|/realm/playgrounds:post}

{generate_parameter_description(/realm/playgrounds:post)}

## Response

{generate_return_values_table|zulip.yaml|/realm/playgrounds:post}

{generate_response_description(/realm/playgrounds:post)}
```

What's the thinking behind the response descriptions sitting after the table of parameters?  

It appears the actual docs have a mix of both styles.  I'll look at the examples and think about this.

Reply:
I think after playing with this, the approach of a note at the end is probably fine. Posted detailed comments on individual items in the other PR.

Reply:
> It appears the organization you've gone with is this:
> 
> ```
> ## Parameters
> 
> {generate_api_arguments_table|zulip.yaml|/realm/playgrounds:post}
> 
> {generate_parameter_description(/realm/playgrounds:post)}
> 
> ## Response
> 
> {generate_return_values_table|zulip.yaml|/realm/playgrounds:post}
> 
> {generate_response_description(/realm/playgrounds:post)}
> ```
> 
> What's the thinking behind the response descriptions sitting after the table of parameters?
> 
> It appears the actual docs have a mix of both styles. I'll look at the examples and think about this.

@timabbott  In addition to the already existing examples, I feel that we would more likely want to talk about common elements about parameters/responses after mentioning them all for the context, and hence the decision. Further, the already existing cases where descriptions occur before seem to be easily adjustable to the other format.

Reply:
Yeah, I think this is fine for now.  It might be better writing style to have these at the top in many cases, but we'll need to rewrite the text to do so, and the goal here is to migrate to a world where it'd be easy to just edit the default/base template anyway.